lib/util/git/modified-cache.spec.ts

```diff
@@ -7,6 +8,8 @@ import {
 } from './modified-cache';
 
 jest.mock('../cache/repository');
+jest.mock('../../logger');
+const logger = mocked(_logger);
```

Comment:
logger is globally mocked, so no need to do it again .

mocked logger can be imported from test/util